cfpb/cfgov-refresh
# Mega menu: Allow more than one featured link in a box

## Changes

- Change markup and styles to support more than one link inside featured link area.

## Testing

1. Pull branch and build.
2. Visit homepage and inspect first featured item in the menu. 
3. Edit the HTML and duplicate the `li` of the featured item and see that two links appear in one box.

## Screenshots

Before:

<img width="370" alt="Screen Shot 2020-03-02 at 6 24 20 PM" src="https://user-images.githubusercontent.com/704760/75727682-86482d80-5cb3-11ea-9e98-29538cce35a4.png">

After:

<img width="365" alt="Screen Shot 2020-03-02 at 6 24 12 PM" src="https://user-images.githubusercontent.com/704760/75727690-89dbb480-5cb3-11ea-968d-484138d33a6c.png">
